packages/Liquid-Core.package/LiquidPollDraftBuilder.class/instance/runPoll.st

```diff
@@ -1,5 +1,6 @@
 button action
 runPoll
-	(LiquidPoll newWithPollDraft: self pollDraft)
-		startWithId: (UIManager default request: 'Enter a poll id' initialAnswer: self generateRandomID)
+self question maxNumberOfChoices ifNil: [self question maxNumberOfChoices: 1].
```

Comment:
zählt ein '' string als Nil?

Reply:
Nein, ein leerer String wäre ein leerer String. Es kann aber nie zu dem Fall kommen, dass in `maxNumberOfChoices` ein leerer String gespeichert ist (vor allem, da `maxNumberOfChoices` eine `Number` ist), das kannst du im Setter `maxNumberOfChoices:` nachlesen. Sollte der vom User eingegebene String leer sein, wird `maxNumberOfChoices` auf `nil` gesetzt.